packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -101,6 +101,13 @@ interface SelectPanelBaseProps {
    */
   footer?: string | React.ReactElement
   showSelectedOptionsFirst?: boolean
+  /**
+   * Whether to opt out of fullscreen behavior on narrow viewports.
+   * When `true`, the panel will maintain its anchored position regardless of viewport size.
+   * When `false`, the panel will go fullscreen on narrow viewports (if feature flag is enabled).
+   * @default undefined (uses feature flag default)
+   */
+  fullScreenOptOut?: boolean
```

Comment:
Really the only thing I'm not sure about is the name 🤔 What would you think of something like `disableFullscreenOnNarrow` or something else that is more clear about what it the prop does and when it does it?